tensorflow_quantum/core/qsim/state_space_test.cc

```diff
@@ -466,6 +466,230 @@ TEST(StateSpaceTest, SampleStateComplexDist) {
   ASSERT_EQ(samples.size(), m);
 }
 
+TEST(StateSpaceTest, ComputeSampledExpectation) {
+  const uint64_t num_qubits(2);
+  const uint64_t q0(0);
+  const uint64_t q1(1);
+  const int m(10000000);
```

Comment:
Could this be safely reduced to 100k instead of 10M ?

Reply:
Lowered precision and num samples